### ISSUE TITLE
Binder links are broken because pangeo binder is down

### ISSUE THREAD

**Issue by rabernat** [pangeo-gallery/pangeo-gallery#67]
Because of https://github.com/pangeo-data/pangeo-binder/issues/195, binder.pangeo.io is down.

We should replace the binder buttons with links to open in JupyterHub. Or at least post a warning somehow.

**Comment by jkingslake** [comment]
Yes! It is a shame to not have these links working in http://gallery.pangeo.io/

So you are suggesting to replace with links to open the NBs in the 2i2c pangeo jupyter hub instead? This will be restricted to pangeo users, right? But I guess that's better than broken links. 

 What's the syntax for the link?
